src/Insolation.jl

```diff
@@ -1,10 +1,10 @@
 module Insolation
 
 using Dates, DelimitedFiles, Interpolations
-using CLIMAParameters
-using CLIMAParameters.Planet
-using CLIMAParameters: AbstractParameterSet
-const APS = AbstractParameterSet
+
+include("Parameters.jl")
+import .Parameters as IP
+const AIP = IP.AbstractInsolationParams
```

Comment:
Is `AIP` used? I saw in other places you’re using `IP.APS`, which I guess is the same thing. I would say it’s cleaner to just use `AIP`, but either way is fine, and we should delete the other one.

Reply:
Yeah, I was a bit inconsistent with this, we can change `APS` to `AIP` (and `IP.AIP`).